CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+# dbt_marketo_source v0.11.0
+[PR #32](https://github.com/fivetran/dbt_marketo/pull/32) and Marketo Source [PR #35](https://github.com/fivetran/dbt_marketo_source/pull/35) include the following updates:
+
+## Feature Updates (includes potentially 🚨 breaking changes 🚨)
+- Ensures that `stg_marketo__lead` (and therefore `marketo__leads`) has and documents the below columns, all [standard](https://developers.marketo.com/rest-api/lead-database/fields/list-of-standard-fields/) fields from Marketo. Previously, peristed all fields found in your `LEAD` source table but only _ensured_ that the `id`, `created_at`, `updated_at`, `email`, `first_name`, `last_name`, and `_fivetran_synced` fields were included. If your `LEAD` table contains the following fields, nothing will change for you. If any of the following default columns are missing from your `LEAD` table, `stg_marketo__lead` will create a NULL version with the proper data type:
```

Comment:
I'm wondering does this bit:
>   - `unsubscribed` -> aliased as `is_unsubscribed` (🚨 breaking change 🚨)
>   - `email_invalid` -> aliased as `is_email_invalid` (🚨 breaking change 🚨)

make this bit not be entirely true?:
> If your `LEAD` table contains the following fields, nothing will change for you.

Would their columns get renamed?